_integration-test/run.sh

```diff
@@ -130,6 +130,42 @@ cat debug.log
 echo '------------------------------------------'
 echo "${_endgroup}"
 
+echo "${_group}Test symbolicator works ..."
+SENTRY_ORG="${SENTRY_ORG:-sentry}"
+SENTRY_PROJECT="${SENTRY_PROJECT:-native}"
+SENTRY_TEAM="${SENTRY_TEAM:-sentry}"
+# First set up a new project if it doesn't exist already
+PROJECT_JSON=$(jq -n -c --arg name "$SENTRY_PROJECT" --arg slug "$SENTRY_PROJECT" '$ARGS.named')
+NATIVE_PROJECT_ID=$(sentry_api_request "api/0/teams/$SENTRY_ORG/$SENTRY_TEAM/projects/" | jq -r '.[]|select(.slug == "'"$SENTRY_PROJECT"'")|.id')
+if [ -z "${NATIVE_PROJECT_ID}" ]; then
+  NATIVE_PROJECT_ID=$(sentry_api_request "api/0/teams/$SENTRY_ORG/$SENTRY_TEAM/projects/" -X POST --data "$PROJECT_JSON" | jq -r '. // null | .id')
+fi
+# Set up sentry-cli command
+SCOPES=$(jq -n -c --argjson scopes '["event:admin", "event:read", "member:read", "org:read", "team:read", "project:read", "project:write", "team:write"]' '$ARGS.named')
+SENTRY_AUTH_TOKEN=$(sentry_api_request "api/0/api-tokens/" -X POST --data "$SCOPES" | jq -r '.token')
+SENTRY_DSN=$(sentry_api_request "api/0/projects/sentry/native/keys/" | jq -r '.[0].dsn.secret')
+# Then upload the symbols to that project (note the container mounts pwd to /work)
+SENTRY_URL="$SENTRY_TEST_HOST" sentry-cli upload-dif --org "$SENTRY_ORG" --project "$SENTRY_PROJECT" --auth-token "$SENTRY_AUTH_TOKEN" windows.sym
+# Get public key for minidump upload
+PUBLIC_KEY=$(sentry_api_request "api/0/projects/sentry/native/keys/" | jq -r '.[0].public')
+# Upload the minidump to be processed, this returns the event ID of the crash dump
+EVENT_ID=$(sentry_api_request "api/$NATIVE_PROJECT_ID/minidump/?sentry_key=$PUBLIC_KEY" -X POST -F 'upload_file_minidump=@windows.dmp' | sed 's/\-//g')
+# We have to wait for the item to be processed
+for i in {0..60..10}; do
+  EVENT_PROCESSED=$(sentry_api_request "api/0/projects/$SENTRY_ORG/$SENTRY_PROJECT/events/" | jq -r '.[]|select(.id == "'"$EVENT_ID"'")|.id')
+  if [ -z "$EVENT_PROCESSED" ]; then
+    sleep $i
+  else
+    break
```

Comment:
So the success case is having our event processed at all? Is it worth implementing the [second half of the test](https://github.com/getsentry/sentry/blob/28f4f36b4adc3d39ea80f2f1864cff8e311b6af3/tests/symbolicator/test_minidump_full.py#L86-L103) to check the integrity of the file?

Reply:
Talked on a call ... I'm good with the limited smoke test here.